galaxy_release_util/point_release.py

```diff
@@ -230,9 +235,33 @@ def add_changelog_item(changes, child):
 def bump_package_version(package: Package, new_version: Version):
     new_content = []
     content = package.setup_cfg.read_text().splitlines()
+    install_requires = False
+    requirements_txt = False
     for line in content:
         if line.startswith("version = "):
             line = f"version = {new_version}"
+        elif line.startswith("install_requires ="):
```

Comment:
This seems kind of weird to execute on all packages. Do you need to first determine if this is the meta package ?

Reply:
(Also any reason we don't symlink the requirements file ?)

Reply:
Also, can we have this in a separate function? A function called `bump_package_version` shouldn't be overwriting package requirements, I think. Besides, we call it twice, and we don't want to have to do this new work twice (or each time we need to update the version number of a package). No to mention ease of testing..

Reply:
> (Also any reason we don't symlink the requirements file ?)

Setuptools does not use this automatically, does it? Apparently since 62.6 you can specify it in setup.cfg with:
```ini
install_requires = file: requirements.txt
```
but this can't be combined with the list syntax we need to specify the galaxy-* subpackages (and the `--index-url` is invalid for setuptools as well).

Reply:
> Also, can we have this in a separate function?

Sure. I did it here because we were already modifying the file here, so you know, didn't want to incur the performance penalty of reading/writing a 1 KB file twice :laughing: but agree it's not in scope for the function.